doc/sphinxext/gen_commands.py

```diff
@@ -106,6 +108,17 @@ def generate_commands_rst(app=None):
     _replace_md5(str(out_fname))
 
 
+def _replace_md5(fname):
+    """Replace a file based on MD5sum."""
+    # adapted from sphinx-gallery
+    assert fname.endswith(".new")
```

Comment:
this won't work for Path objects, is that OK?

Reply:
Must be, this is just copy-pasted :slightly_smiling_face: